crm_brand/models/crm_team.py

```diff
@@ -0,0 +1,15 @@
+# Copyright 2021 Yves Goldberg
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+
+
+class CrmTeam(models.Model):
+
+    _name = 'crm.team'
+    _inherit = ['crm.team', 'res.brand.mixin']
+
+    brand_id = fields.Many2one(
```

Comment:
please consider that brand field is already added to crm.team in sale_brand module, see https://github.com/OCA/brand/blob/12.0/sale_brand/models/crm_team.py#L10

there is a PR to fast forward it to 13.0 #52

Reply:
oh great. so this PR is not needed and I close it. right?

Reply:
yes If you don't need brand on crm.lead